cli_tools/gce_onestep_image_import/onestep_importer/aws_args.go

```diff
@@ -97,9 +101,8 @@ func (args *awsImportArguments) validateAndPopulate(populator param.Populator) e
 		return err
 	}
 
-	networkNotUsed, subnetNotUsed := "", ""
 	err = populator.PopulateMissingParameters(args.gcsProjectPtr, args.clientID, &args.gcsZone,
-		&args.gcsRegion, &args.gcsScratchBucket, "", &args.gcsStorageLocation, &networkNotUsed, &subnetNotUsed)
+		&args.gcsRegion, &args.gcsScratchBucket, "", &args.gcsStorageLocation, &args.gcsNetwork, &args.gcsSubnet)
```

Comment:
Can we consider adding a test for this case?

Reply:
+1. Add a e2e.